docs/en_US/TrainingService/PaiMode.md

```diff
@@ -19,6 +19,8 @@ maxExecDuration: 3h
 maxTrialNum: 100
 # choice: local, remote, pai
 trainingServicePlatform: pai
+# search space file
+searchSpacePath: nni_search_space.json
```

Comment:
It seems all the name of search space file in nni examples all called "search_space.json".

So I suggest "nni_search_space.json" -> "search_space.json"

Reply:
ok~